ansible_collections/arista/avd/roles/eos_designs/docs/how-to/wan.md

```diff
@@ -0,0 +1,493 @@
+      wan_vni: 1
+  policies:
+    - name: PROD-AVT-POLICY  # (3)!
+      default_virtual_topology: # (4)!
+        path_groups:
+          - names: [INET]
+            preference: preferred
+          - names: [MPLS]
+            preference: alternate
+      application_virtual_topologies:
+        - application_profile: VOICE # (5)!
+          lowest_hop_count: true
+          path_groups:
+            - names: [MPLS]
+              preference: preferred
+            - names: [INET]
+              preference: alternate
+          constraints: # (6)!
+            delay: 120
+            jitter: 20
+            loss_rate: 0.3
+          id: 2 # (7)!
+        - application_profile: VIDEO
+          path_groups:
+            - names: [MPLS, LTE]
+              preference: preferred
+            - names: [INET]
+              preference: alternate
+          constraints:
+            loss_rate: 42.0
+          id: 4
+        # This will not be rendered on devices without MPLS
+        - application_profile: MPLS-ONLY # (8)!
+          path_groups:
+            - names: [MPLS]
+              preference: preferred
+          id: 5
+```
+
+1. Assign the `PROD-AVT-POLICY` to the `PROD` VRF, multiple VRFs can use the same policy.
+2. The `default` VRF will use the AVD auto-generated `DEFAULT-POLICY` as no policy is set.
+3. Define the `PROD-AVT-POLICY`,
+4. `default_virtual_topology` is used to configure the default match in the policy.
+    In this case, default traffic will use INET path-group first and MPLS as backup.
+5. This list element configures the policy to apply to traffic the `VOICE` application profile.
+   This block of configuration will configure the Load Balance policy, the match statement in the policy (in `router path-selection` for AutoVPN or `router adaptive-virtual-topology` for CV-Pathfinder) and for CV-Pathfinder, the AVT profile.
+   The application profile must be defined under `application_classification.application_profiles`.
+   The Load Balance policy will favor direct path over multihop path (`lowest_hop_count`).
+   MPLS is prefered over the INET path-group.
+6. The contraints are applied on the load-balance policy. If the delay, jitter or loss rate of a given path-selection path
+   exceeds the defined criteria, the path is avoided and the remaining ECMP paths are used or the next path in line meeting
+   the constraints is used.
+7. The `id` is used as the AVT profile ID in the configuration.
+8. The `MPLS-ONLY` application profile won't be rendered on devices not having this path-group.
+
+### LAN Designs
+
+!!! danger "VRF default routing"
+
+    An important design point to keep in mind is that the current CV Pathfinder and AutoVPN solutions require the Dps1 interface to be in VRF default.
+    This implies that all the WAN interfaces live in default VRF and the LAN interfaces also have subnets in the default VRF.
+
+!!! warning
+
+    AVD does not yet configure any route-map to filter potential routes received
```

Comment:
well all this is not really relevant as long as we don't support dynamic routing on the WAN interfaces...

Reply:
removing - we can add back later once we do support it